Update filelogreceiver/README.md

djaglowski opened the pull request:
Resolves #2335

djaglowski committed:
Update README.md

tigrannajaryan commented on receiver/filelogreceiver/README.md:
```diff
@@ -1,29 +1,63 @@
+### Operators
 
-## Operator Basics
+Each operator performs a simple responsibility, such as parsing a timestamp or JSON. Chain together operators to process logs into a desired format.
```
Only "parser" and "general purpose" operators make sense to be used, right? Maybe mention that here.

djaglowski commented on receiver/filelogreceiver/README.md:
```diff
@@ -1,29 +1,63 @@
+| `include`              | required         | A list of file glob patterns that match the file paths to be read                                                  |
+| `exclude`              | []               | A list of file glob patterns to exclude from reading                                                               |
+| `start_at`             | `end`            | At startup, where to start reading logs from the file. Options are `beginning` or `end`                            |
+| `write_to`             | $                | The record [field](/docs/types/field.md) written to when creating a new log entry                                  |
```
`$` can be used as shorthand for `$record`, but this does not appear to be documented. 

In any case, I will just update this to the expanded form.

djaglowski replied:
I'm also going to clean up the `fields` documentation a bit. Will link back here shortly.

djaglowski replied:
https://github.com/open-telemetry/opentelemetry-log-collection/pull/21

djaglowski committed:
Update README.md

Fix a broken link, clarify default for `write_to`, and clarify usage of `operators`.
